src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -173,7 +173,7 @@ describe('Alerts (Unit)', () => {
       await app.close();
     });
 
-    it('returns 403 (Forbidden) for valid signature/invalid payload', async () => {
+    it('returns 404 (Not found) for valid signature/invalid payload', async () => {
```

Comment:
I had to change this expectation because the route won't be mounted if the `alerts` feature is not active in the configuration, and therefore the service returns a 404 instead. 

If we agree on this approach (enabling/disabling the `alerts` modules via configuration), we can go further and remove `AlertsRouteGuard` as it won't be used anymore.